types/error/InvalidFieldError.raml

```diff
@@ -12,3 +12,4 @@ properties:
         type: any
     allowedValues?:
         type: array
+        items: string
```

Comment:
Here I would expect `any` and if you don't specify the `items` facet, `any` should be the fallback. That's why I'm a bit confused about this PR: May be @jenschude can explain the problem a bit?

Reply:
Oh right :D we were running into a strange issue in the JVM code gen and thought it may have come from undeclared items types. This one should really be not defined as it could be anything